html/elements/input/datetime-local.json

```diff
@@ -18,11 +18,10 @@
                 "version_added": "12"
               },
               "firefox": {
-                "version_added": false,
-                "notes": "See <a href='https://bugzil.la/888320'>bug 888320</a> and <a href='https://wiki.mozilla.org/TPE_DOM/Date_time_input_types'>TPE DOM/Date time input types</a>."
+                "version_added": "93"
               },
               "firefox_android": {
-                "version_added": true
+                "version_added": "93"
```

Comment:
Just so it doesn't get lost with the other discussion. This has existed in Firefox Android forever (for all modern intents and purposes). `true` was a better representation of the situation that support for this browser isn't something to worry about.

Reply:
You're absolutely right, I've changed it back. FF Android is currently on version 90, so changing this to 93 would not make sense.